examples/arduino-mega2560/Cargo.toml

```diff
@@ -5,6 +5,9 @@ authors = ["Rahix <rahix@rahix.de>"]
 edition = "2021"
 publish = false
 
+[build-dependencies.proc-macro2]
+version = "=1.0.79"
+
```

Comment:
Hm, I guess this is unavoidable at this time...  I think the best solution is for me to just add the pinning to all example crates for now.  I'll do this and then you can rebase your branch on top.

Reply:
Added the pinning in #544.  Please rebase on latest `main`.

Reply:
removed my duplicate dep after rebase